aspnetcore/tutorials/razor-pages/razor-pages-start.md

```diff
@@ -273,17 +417,19 @@ At the end of this tutorial, you'll have a working Razor Pages web app that you'
 
   ![Home or Index page](razor-pages-start/_static/homeGDPR2.2.png)
 
-  The following image shows the app after you give consent to tracking:
+  The following image shows the app after consent to tracking is provided:
 
   ![Home or Index page](razor-pages-start/_static/home2.2.png)
   
 # [Visual Studio Code](#tab/visual-studio-code)
 
   [!INCLUDE[](~/includes/trustCertVSC.md)]
 
-* Press **Ctrl-F5** to run without the debugger.
+* Press <kbd>Ctrl+F5</kbd> to run without the debugger.
+
+  Launching the app with <kbd>Ctrl+F5</kbd> (non-debug mode) allows you to make code changes, save the file, refresh the browser, and see the code changes. Many developers prefer to use non-debug mode to quickly launch the app and view changes.
 
-  Visual Studio Code starts [Kestrel](xref:fundamentals/servers/kestrel), launches a browser, and navigates to `http://localhost:5001`. The address bar shows `localhost:port#` and not something like `example.com`. That's because `localhost` is the standard hostname for  local computer. Localhost only serves web requests from the local computer.
+  Visual Studio Code starts [Kestrel](xref:fundamentals/servers/kestrel), launches a browser, and goes to `http://localhost:5001`. The address bar shows `localhost:port#` and not something like `example.com`. That's because `localhost` is the standard hostname for  local computer. Localhost only serves web requests from the local computer.
```

Comment:
```suggestion
  Visual Studio Code starts [Kestrel](xref:fundamentals/servers/kestrel), launches a browser, and goes to `http://localhost:5001`. The address bar shows `localhost:port#` and not something like `example.com`. That's because `localhost` is the standard hostname for the local computer. Localhost only serves web requests from the local computer.
```